### ISSUE TITLE
Local connection does not reconnect for state/attribute updates

### ISSUE THREAD

**Issue by peledzius** [jnimmo/pyIntesisHome#44]
In HA when my device is dropped from WiFi or I reboot router - entity state and attributes aren't updated anymore after device comes back online, but I still can control my unit: turn on/off, adjust temperature, change fan mode, etc. HA restart solves the problem until next connection drop. Logs do not show anything about connection drop. I assume local connection control missing reconnection logic for state updates.

**Comment by ldsmovil** [comment]
Im with latest versión of core and supervisor in HA. No update status, and climate entity is marked as unavailable. i cant use the climate from HA.

With AC Cloud APP works without problems. Tried to reconnect Intesishome unit, reboot router, restart HA several times....

Logger: pyintesishome
Source: runner.py:119
First occurred: 17:34:08 (1 occurrences)
Last logged: 17:34:08

pyIntesisHome lost connection to the IntesisHome server. Exception: 0 bytes read on a total of undefined expected bytes

**Comment by jnimmo** [comment]
I've bumped pyIntesisHome to 1.8.3, and pushed an update to the HACS addon https://github.com/jnimmo/hass-intesishome to 1.0.2, this will hopefully result in some more logging if an unexpected error occurs, and fingers crossed the update task loop won't die in the event of an exception which may have been the case before.

**Comment by jnimmo** [comment]
> Reference

This sounds like a bit of a different issue, please check out the Home Assistant forums, at the very least debug logging is required to be able to look into this any further - please set this up in your configuration.yaml file like so:
```
logger:
  default: info
  logs:
    homeassistant.components.intesishome: debug
    custom_components.intesishome: debug
    pyintesishome: debug
```

**Comment by jnimmo** [comment]
This has been resolved in 1.0.5 after discovering some exception handling issues, please re-open issue if it continues.
